src/cms/views/events/event_list_view.py

```diff
@@ -5,7 +5,7 @@
 from django.contrib.auth.mixins import LoginRequiredMixin, PermissionRequiredMixin
 from django.shortcuts import render, redirect
 from django.utils.decorators import method_decorator
-from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext as _, ugettext_lazy
```

Comment:
```suggestion
from django.utils.translation import ugettext as _
```
You only need `ugettext_lazy` in forms and models. In views, you can use `ugettext` without problems.

Reply:
Sorry, but this is wrong. The reason you need lazy translation is, that you just just want to store a reference to the string and translate that as needed (when the template is rendered: https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#lazy-translations). You can see what I mean when you do the following:

- Use ugettext as you suggest and save these in context (I propose the page menu, there you got delete/archive buttons)
- Choose english language in the upper right corner
- Click for example the delete button for arbitrary page
- Then you see that german messages are rendered in the dialog instead of english as you would expect

Reply:
Oh sorry! I didn't realize the fact that the context is defined as class attribute in comparison to other strings in the view's `get()`-method. You right, in this case the lazy method is required in the view.